src/main/java/io/devlabs/keytree/domains/company/domain/Authority.java

```diff
@@ -0,0 +1,5 @@
+package io.devlabs.keytree.domains.company.domain;
+
+public enum Authority {
+    ADMIN
+}
```

Comment:
```suggestion
public enum UserRole {
    MASTER("MASTER");
    ADMIN("ROLE_ADMIN");
    USER("ROLE_USER"),

    private final String authority;

    UserRole(String authority) {
        this.authority = authority;
    }

    public String getAuthority() {
        return authority;
    }
}
```
- SUPER_ADMIN, MASTER, ROOT 중에 어떤걸로 선택할지 조금 고민해보면 좋을것 같아요.
- 자바에서 enum은 많이 사용되는 문법이니 기본적인 것들을 조금 공부하고 쓰시면 좋을것 같아요.

Reply:
MASTER, ADMIN, USER가 더욱 직관적일 것 같아서 제안 주신 코드로 적용했습니다.